glide-core/redis-rs/redis/src/cluster_async/mod.rs

```diff
@@ -2139,6 +2204,319 @@ where
             .map_err(|err| (OperationTarget::Node { address }, err))
     }
 
+    // This function handles commands with routing info of MultiSlot (like MSET or MGET), creates sub-commands for the matching slots and add it to the correct pipeline
+    async fn handle_multi_slot_routing(
```

Comment:
The name of the function should declare that it's only relevant for pipelines, the current name is misleading